templates/openstackconfiggenerator/config/16.2/nic/net-config-multi-nic.role.j2.yaml

```diff
@@ -119,68 +131,71 @@ resources:
                 routes:
                   list_concat_unique:
                     - get_param: ControlPlaneStaticRoutes
-{{`{%- if role.default_route_networks is not defined or 'ControlPlane' in role.default_route_networks %}`}}
+{%- if role.default_route_networks is not defined or 'ControlPlane' in role.default_route_networks %}
                     - - default: true
                         next_hop:
                           get_param: ControlPlaneDefaultRoute
-{{`{%- endif %}`}}
-{{- $nic_counter := 2 }}
-{{- range $netid, $net := .Role.Networks }}
-{{- if and (ne $net.Name "External") (ne $net.Name "Tenant") (ne $net.NameLower "ctlplane") }}
-{{- $nic_counter = add $nic_counter 1 }}
+{%- endif %}
+{%- set nics_used = [2] %}
```

Comment:
This is used at the end of the loop, but I'm sure why it is needed when we use `loop.index + 2` (seemingly) instead.  Could you explain what is happening here?

Reply:
ah we can remove that. It comes from the default multi nic role template[1] where ad the end they add nics for the ComputeOvsDpdk role[2]. In our case the `loop.index + 2` should be enough. I'll update it.

[1] https://github.com/openstack/tripleo-heat-templates/blob/stable/train/network/config/multiple-nics-vlans/role.role.j2.yaml#L137
[2]https://github.com/openstack/tripleo-heat-templates/blob/stable/train/network/config/multiple-nics-vlans/role.role.j2.yaml#L206